examples/quickstart-pytorch/README.md

```diff
@@ -72,3 +74,41 @@ python3 client.py --partition-id 1
 ```
 
 You will see that PyTorch is starting a federated training. Look at the [code](https://github.com/adap/flower/tree/main/examples/quickstart-pytorch) for a detailed explanation.
+
+______________________________________________________________________
+
+## Run Federated Learning with PyTorch and `Flower Next`
+
+### 1. Start the long-running Flower server (SuperLink)
+
+```bash
+flower-superlink --insecure
+```
+
+### 2. Start the long-running Flower clients (SuperNodes)
+
+Start 2 long-running Flower clients in 2 separate terminal windows, using:
+
+```bash
+flower-client-app client:app --insecure
+```
+
+### 3. Run the Flower App
+
+With both the long-running server (SuperLink) and two clients (SuperNode) up and running, we can now run the actual Flower App:
+
+```bash
+flower-server-app server:app --insecure
+```
+
+Or, to try the workflow example, run:
+
+```bash
+flower-server-app server_workflow:app --insecure
+```
+
+Or, to try the custom server function example, run:
+
+```bash
+flower-server-app server_custom:app --insecure
+```
```

Comment:
This was only relevant for the `exampes/app-pytorch`

Reply:
Ah! I see.